manticore/platforms/linux.py

```diff
@@ -2408,6 +2481,47 @@ def sys_open(self, buf, flags, mode):
 
         return rv
 
+    def sys_openat(self, dirfd, buf, flags, mode):
+        '''
+        A version of openat that includes a symbolic path and symnbolic directory file descriptor
+
+        :param dirfd: directory file descriptor
+        :param buf: address of zero-terminated pathname
+        :param flags: file access bits
+        :param mode: file permission mode
+        '''
+
+        if issymbolic(dirfd):
+            logger.debug("Ask to read from a symbolic directory file descriptor!!")
```

Comment:
Should not we constraint the dirfd so it can only _range_ into the already opened fd set (+ 1 unopened)?